packages/insomnia-app/app/plugins/context/store.js

```diff
@@ -2,7 +2,16 @@
 import type { Plugin } from '../index';
 import * as models from '../../models';
 
-export function init(plugin: Plugin) {
+export type PluginStore = {
+  hasItem(string): Promise<boolean>,
+  setItem(string, string): Promise<void>,
+  getItem(string): Promise<boolean | null>,
```

Comment:
This signature doesn't seem to match the function

```suggestion
  getItem(string): Promise<string | null>,
```

Reply:
good catch 😅